src/modules/renderComment.js

```diff
@@ -0,0 +1,130 @@
+import { baseUrl, involvementApi } from '../../config/keys.js';
+import getItemComments from './getCommentItems.js';
+
+let commentList = [];
+
+export const updateCommentCount = (comment, commentsHeader, add) => {
+  if (!comment.error && add !== true) {
+    commentsHeader.innerHTML = `comments (${comment.length})`;
```

Comment:
- [ ] The same thing is happening here, you are utilizing the incoming data from the API to display the comments count. Please create a separate function as discussed earlier to handle the counting. 🥸️